pr_watch/models.py

```diff
@@ -86,49 +93,33 @@ def get(self, *args, **kwargs):
         return super().get(*args, **kwargs)
 
 
-class VersionControlInstanceMixin(models.Model):
+class WatchedPullRequest(models.Model):
     """
-    Instances linked to a VCS, such as git
+    List of all watched pull requests, with ID of the Instance created for that PR, if any
```

Comment:
@itsjeyd No, each instance of this model represents a single PR, and one OpenEdXInstance associated with the PR.

I changed the docstring since I was trying to describe the table, not each individual instance/model, and that isn't best practice here.